drivers/backup/portworx/portworx.go

```diff
@@ -0,0 +1,477 @@
+package portworx
+
+import (
+	"context"
+	"fmt"
+
+	api "github.com/portworx/px-backup-api/pkg/apis/v1"
+	"github.com/portworx/sched-ops/k8s/core"
+	"github.com/portworx/torpedo/drivers/backup"
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/drivers/volume"
+	"github.com/portworx/torpedo/drivers/volume/portworx/schedops"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	// import ssh to invoke init
+	_ "github.com/portworx/torpedo/drivers/node/ssh"
+	// import scheduler k8s
+	_ "github.com/portworx/torpedo/drivers/scheduler/k8s"
+	// import portworx volume
+	_ "github.com/portworx/torpedo/drivers/volume/portworx"
+)
+
+const (
+	driverName            = "pxb"
+	pxbRestPort           = 10001
+	defaultPxbServicePort = 10002
+	pxbServiceName        = "px-backup"
+	pxbNameSpace          = "px-backup"
+	schedulerDriverName   = "k8s"
+	nodeDriverName        = "ssh"
+	volumeDriverName      = "pxd"
+	awsAccessKey          = "CT6R80D3ST0VW9NY6HYP"
+	awsSecretKey          = "a0V6dPqu8C26KbAsa9qsIrfhsbvyGjjPPmZN2qD4"
+)
+
+type pxbackup struct {
+	clusterManager         api.ClusterClient
+	backupLocationManager  api.BackupLocationClient
+	cloudCredentialManager api.CloudCredentialClient
+	backupManger           api.BackupClient
+	restoreManager         api.RestoreClient
+	backupScheduleManager  api.BackupScheduleClient
+	schedulePolicyManager  api.SchedulePolicyClient
+	organizationManager    api.OrganizationClient
+	healthManager          api.HealthClient
+
+	schedulerDriver scheduler.Driver
+	nodeDriver      node.Driver
+	volumeDriver    volume.Driver
+	schedOps        schedops.Driver
+	refreshEndpoint bool
+	token           string
+}
+
+func (d *pxbackup) String() string {
```

Comment:
```suggestion
func (p *pxbackup) String() string {
```
The convention is to use the first letter of the struct for the instance variable